docs/frameworks/sveltekit.md

```diff
@@ -4,212 +4,252 @@ title: SvelteKit | Frameworks
 
 # SvelteKit
 
+::: warning
+From SvelteKit version `1.0.0-next.359+`, SvelteKit is just another Vite plugin, and latest versions will also require you to update your application to use Vite ^3.1.0 and Node 16.14+.
```

Comment:
359 is too old because it's not a sequential plugin
```suggestion
We recommend you use the latest version of SvelteKit. The latest versions will also require you to update your application to use Vite ^3.1.0 and Node 16.14+.
```